### PR TITLE
Removing ActiveStorage Integration

### LINKED ISSUE #1144: Remove ActiveStorage layer between Uploads and AWS S3
We want to simplify the code that works with uploads `precuration_uploads` and `pre_curation_uploads_fast` exist.  The first is utilizing ActiveStorage and the second interfaces directly with AWS.

We moved away from ActiveStorage when  we realized that it was very slow for large numbers of files.  What we want is for the uploads to act like the readme does and directly upload via the S3QueryService.

- [ ] Move all configuration from the active storage `storage.yml` into the s3 configuration file and remove `active_storage_configuration` from the S3QueryService
- [ ] Remove precuration_uploads from the work and update the code to upload via the S3QueryService.upload_file (The readme does this already for an example)
- [ ] Add/ clean up tests as needed

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #1144

### CODE REVIEW

File: app/services/s3_query_service.rb [hunk 1]
```diff
@@ -122,14 +122,17 @@ def find_s3_file(filename:)
...
   def client_s3_files(reload: false, bucket_name: self.bucket_name, prefix: self.prefix, ignore_directories: true)
+    return [] unless model.persisted?
+
     @client_s3_files = nil if reload # force a reload
     @client_s3_files ||= begin
       start = Time.zone.now
       resp = client.list_objects_v2({ bucket: bucket_name, max_keys: 1000, prefix: prefix })
       resp_hash = resp.to_h
       objects = parse_objects(resp_hash, ignore_directories: ignore_directories)
-      objects += parse_continuation(resp_hash, bucket_name: bucket_name, prefix: prefix, ignore_directories: ignore_directories)
```
Comment (hectorcorrea) [inline app/services/s3_query_service.rb:131]: Why was this line removed? I seem to remember that this is what fetches when there are more than 1000 files. Are we doing this differently now?

File: app/models/work.rb [hunk 2]
```diff
@@ -377,6 +380,53 @@ def current_transition
...
+    alias append <<
+  end
+
+  def pre_curation_uploads_fast
+    s3_resources = s3_query_service.client_s3_files.sort_by(&:filename)
+    UploadsProxy.new(model: self, values: s3_resources)
+  end
+  alias pre_curation_uploads pre_curation_uploads_fast
+
+  # Deprecate
```
Comment (jrgriffiniii) [inline app/models/work.rb:420]: This still needs to be removed, I apologize.

File: app/models/work.rb [hunk 3]
```diff
@@ -377,54 +382,57 @@ def current_transition
...
+    alias append <<
+  end
+
+  # Accesses the S3 file uploads for Works in pre-curation/preapproved states
+  # @return [Array<S3File>]
+  def pre_curation_uploads
+    values = s3_query_service.client_s3_files.sort_by(&:filename)
 
-    save_new_attachments(new_attachments: new_attachments)
+    UploadsProxy.new(model: self, values: values)
```
Comment (jrgriffiniii) [inline app/models/work.rb:425]: This is breaking the test suite (due to an absence of mocks for HTTP responses)
